tasks/DroneFlightControlTask.hpp

```diff
@@ -1,46 +1,62 @@
 /* Generated from orogen/lib/orogen/templates/tasks/Task.hpp */
 
-#ifndef DRONE_DJI_SDK_TASK_TASK_HPP
-#define DRONE_DJI_SDK_TASK_TASK_HPP
-
-#include "drone_dji_sdk/TaskBase.hpp"
-
-namespace drone_dji_sdk{
-
-    /*! \class Task
+#ifndef DRONE_DJI_SDK_DRONEFLIGHTCONTROLTASK_TASK_HPP
+#define DRONE_DJI_SDK_DRONEFLIGHTCONTROLTASK_TASK_HPP
+
+#include "drone_dji_sdk/DroneFlightControlTaskBase.hpp"
+#include "drone_dji_sdkTypes.hpp"
+#include <gps_base/UTMConverter.hpp>
+#include <gps_base/BaseTypes.hpp>
+#include "djiosdk/dji_telemetry.hpp"
+#include "djiosdk/dji_vehicle.hpp"
+#include "djiosdk/dji_status.hpp"
+#include <pthread.h>
+#include <cmath>
+#include <semaphore.h>
+#include "stdint.h"
+#include <stdlib.h>
+#include <stdexcept>
+#include <sys/time.h>
+#include <fcntl.h>
+#include <termios.h>
+#include <unistd.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+#include <arpa/inet.h>
```

Comment:
I'm fairly sure you use none of these in here. Let's try to keep the list of includes in the .hpp to a minimum

Reply:
kkk I forgot to remove, sorry